workers/prettier-worker.ts

```
@@ -116,6 +160,28 @@ async function loadPrettier(): Promise<boolean> {
 }
 
+/**
+ * Resolves and merges prettier configuration.
+ *
+ * @param filePath - Optional file path for config resolution.
+ * @param configOverride - Optional config overrides.
+ * @returns Promise resolving to merged config.
+ */
+async function resolveConfig(
+	filePath: string | undefined,
+	configOverride: PrettierOptions | undefined,
```

ESLint

[error] 'PrettierOptions' is an 'error' type that acts as 'any' and overrides all other types in this union type.

(ts/no-redundant-type-constituents)